opentelemetry-distro/src/opentelemetry/distro/__init__.py

```diff
@@ -52,7 +52,8 @@ def _get_exporter_names() -> Sequence[str]:
 
     if (
         trace_exporters is not None
-        or trace_exporters.lower().strip() != "none"
+        and trace_exporters != ""
```

Comment:
Can we just do `if trace_exporters and trace_exporters.lower().strip() != "none"`?

Reply:
That's right. 👍